readme.md

```diff
@@ -32,6 +32,13 @@ var nsAttr = doc.documentElement.getAttributeNS('./lite','x')
 console.info(nsAttr)
 console.info(doc)
 ```
+
+Note: in Typescript and ES6 you can use the import approach, as follows:
```

Comment:
```suggestion
Note: a more modern import approach is possible in Typescript and ES6, as follows:
```

Reply:
I can make the changes. Had just been trained by other projects with the 'fixes' prefix to allow auto closing of tickets, but I understand each project is different. I'll need to make a new PR to remove that and close this one.

Reply:
I went ahead with the merge, just moved your fixes note from the subject into the comments in the commit message. Your note is shorter so I just kept it as-is. Thanks again!